README.md

```diff
@@ -29,7 +29,10 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+def offer_rose (person)
+  offer_rose(young prince)
+  puts "Would you take this rose" + person, "in exchange for giving an old beggar woman shelter from the bitter cold?"
+end
 ```
 
```

Comment:
You'd call `offer_rose` outside the function definition.` offer_rose("Bob Pizza")`. Also, you'd pass in a string since you don't have a variable defined for what I think you meant is one variable `young_prince`. `young` and `prince` as written now, looks like you're trying to pass in two different variables.